precode.go

```diff
@@ -55,7 +71,16 @@ func main() {
 	var wg sync.WaitGroup
 
 	// 4. Собираем числа из каналов outs
-	// ...
+	for i, out := range outs {
+		wg.Add(1)
+		go func(i int, out <-chan int64) {
```

Comment:
Лучше называть переменные по-другому, чтобы точно было понято откуда она взята
```suggestion
		go func(i int, in <-chan int64) {
```